controllers/runner_controller.go

```diff
@@ -1123,3 +1148,21 @@ func removeFinalizer(finalizers []string, finalizerName string) ([]string, bool)
 
 	return result, removed
 }
+
+func workVolumePresent(items []corev1.Volume) (bool, int) {
```

Comment:
I could make this fn type agnostic, that it accepts `items []corev1.Volume` and `items []corev1.VolumeMount` like the fn below